From 1cb5fc9e6e15c5fb97e60904c326afa979494fa9 Mon Sep 17 00:00:00 2001
From: "J. German Rivera" <German.Rivera@freescale.com>
Date: Wed, 11 Mar 2015 22:52:54 -0500
Subject: [PATCH 025/451] staging: fsl-mc: Corrected email addresses in TODO
 file

Signed-off-by: J. German Rivera <German.Rivera@freescale.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
(cherry picked from commit 11dd0360cf882182d780eb9941cde404a4e21034)
[Original patch from FSL LS2085 SDK EAR4.0]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-mc/TODO |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-mc/TODO b/drivers/staging/fsl-mc/TODO
index 49ebfd9..d78288b 100644
--- a/drivers/staging/fsl-mc/TODO
+++ b/drivers/staging/fsl-mc/TODO
@@ -8,5 +8,6 @@
 * Add at least one device driver for a DPAA2 object (child device of the
   fsl-mc bus).
 
-Please send any patches to Greg Kroah-Hartman <greg@kroah.com>,
-devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org
+Please send any patches to Greg Kroah-Hartman <gregkh@linuxfoundation.org>,
+german.rivera@freescale.com, devel@driverdev.osuosl.org,
+linux-kernel@vger.kernel.org
-- 
1.7.5.4

